crypto/replay/rdbx.c

```diff
@@ -259,16 +253,17 @@ unsigned long srtp_rdbx_get_window_size (const srtp_rdbx_t *rdbx)
  * srtp_rdbx_check(&r, delta) checks to see if the srtp_xtd_seq_num_t
  * which is at rdbx->index + delta is in the rdb
  */
-srtp_err_status_t srtp_rdbx_check (const srtp_rdbx_t *rdbx, int delta)
+srtp_err_status_t srtp_rdbx_check(const srtp_rdbx_t *rdbx, int delta)
 {
-
-    if (delta > 0) {     /* if delta is positive, it's good */
+    if (delta > 0) { /* if delta is positive, it's good */
         return srtp_err_status_ok;
     } else if ((int)(bitvector_get_length(&rdbx->bitmask) - 1) + delta < 0) {
         /* if delta is lower than the bitmask, it's bad */
         return srtp_err_status_replay_old;
-    } else if (bitvector_get_bit(&rdbx->bitmask,
-                                 (int)(bitvector_get_length(&rdbx->bitmask) - 1) + delta) == 1) {
+    } else if (bitvector_get_bit(
+                   &rdbx->bitmask,
+                   (int)(bitvector_get_length(&rdbx->bitmask) - 1) + delta) ==
+               1) {
```

Comment:
We might be able to remove this lonely one here.  I think that function call returns either a 0 or 1, and I think the aim is to just check for a non-zero response.

Reply:
Not sure if there is a better way of doing it, maybe if it was changed to if (1 == bitvector...... 
Would think it was possible to do it on 2 lines instead of 4 but since you approved I am going to leave it as is. Feel free to play with clang-format to get this better!